user/xournalpp/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "xournalpp"
+pkgver = "1.2.3"
+pkgrel = 0
+archs = ["aarch64", "x86_64"]
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "pkgconf",
+    "gettext",
+    "ninja",
+    "help2man",
+    "gtksourceview4-devel",
```

Comment:
makedepends, not hostmakedepends

this list is also not sorted

Reply:
oops slipped